src/App/Pages/Settings/LoginPasswordlessRequestsListViewModel.cs

```diff
@@ -34,8 +34,9 @@ public LoginPasswordlessRequestsListViewModel()
             PageTitle = AppResources.PendingLogInRequests;
             LoginRequestsList = new ObservableRangeCollection<PasswordlessLoginResponse>();
 
-            AnswerRequestCommand = new AsyncCommand<PasswordlessLoginResponse>((request) => PasswordlessLoginAsync(request),
-               onException: ex => HandleException(ex));
+            AnswerRequestCommand = new AsyncCommand<PasswordlessLoginResponse>(PasswordlessLoginAsync,
+               onException: ex => HandleException(ex),
+                allowsMultipleExecutions: false);
```

Comment:
Fix spacing so `onException` and `allowsMultipleExecutions` are vertically aligned